docs/src/components/guidelines/guideline.module.css

```diff
@@ -8,15 +8,8 @@
   height: 100%;
   padding: var(--psLayoutSpacingXLarge);
   position: relative;
-
-  &.dark {
-    background: var(--psColorsBackgroundDark3);
-    color: var(--psColorsTextIconHighOnDark);
-  }
-
-  &.light {
-    background: var(--psColorsBackgroundLight2);
```

Comment:
So this isn't a 1:1 switch it's both `appColorsBackgroundHi` and `appColorsBackgroundMed`

Reply:
True enough. I made the call for convenience. And I think it looks great. Happy to change it back if needed.

<img width="763" alt="Screen Shot 2020-10-07 at 10 58 50 AM" src="https://user-images.githubusercontent.com/552044/95363069-2f7b8580-088c-11eb-8df1-54c58acb5dae.png">
<img width="745" alt="Screen Shot 2020-10-07 at 10 58 44 AM" src="https://user-images.githubusercontent.com/552044/95363073-30acb280-088c-11eb-8ead-1cc565404576.png">